doc/user/bgp.rst

```diff
@@ -2381,13 +2381,15 @@ The following commands can be used in route maps:
    the one of BGP communities value match to the one of community alias value in
    community alias, it is match.
 
-.. clicmd:: match community WORD exact-match [exact-match]
+.. clicmd:: match community WORD [<exact-match|any>]
```

Comment:
Also, this should be modified for match large-community?

Reply:
right